script/production/mainnet/oracleAdapters/Euler V2 Oracles (Ethereum) - RedStone Core (Pull).csv

```diff
@@ -1,135 +1,135 @@
+ezETH,ETH_FUNDAMENTAL,RedStone Core,Yes,Yes,RedstoneCoreOracle,0xbf5495Efe5DB9ce00f80364C8B423567e58d2110,0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2,ezETH_FUNDAMENTAL,8,120
+MERL,USD,RedStone Core,No,No,RedstoneCoreOracle,N/A,0x0000000000000000000000000000000000000348,MERL,8,120
+SolvBTC_MERLIN,USD,RedStone Core,No,No,RedstoneCoreOracle,N/A,0x0000000000000000000000000000000000000348,SolvBTC_MERLIN,8,120
+SolvBTC_MERLIN,BTC,RedStone Core,No,No,RedstoneCoreOracle,N/A,0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB,SolvBTC_MERLIN/BTC,8,120
+SolvBTC_MERLIN,BTC_TWAP_60,RedStone Core,No,No,RedstoneCoreOracle,N/A,0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB,SolvBTC_MERLIN/BTC-TWAP-60,8,120
+KES,USD,RedStone Core,No,No,RedstoneCoreOracle,N/A,0x0000000000000000000000000000000000000348,KES,8,120
+PHP,USD,RedStone Core,No,No,RedstoneCoreOracle,N/A,0x0000000000000000000000000000000000000348,PHP,8,120
+COP,USD,RedStone Core,No,No,RedstoneCoreOracle,N/A,0x0000000000000000000000000000000000000348,COP,8,120
+INR,USD,RedStone Core,No,No,RedstoneCoreOracle,N/A,0x0000000000000000000000000000000000000348,INR,8,120
+bsdETH,USD,RedStone Core,No,No,RedstoneCoreOracle,N/A,0x0000000000000000000000000000000000000348,bsdETH,8,120
+bsdETH,ETH,RedStone Core,No,No,RedstoneCoreOracle,N/A,0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2,bsdETH/ETH,8,120
+ZK,USD,RedStone Core,No,No,RedstoneCoreOracle,N/A,0x0000000000000000000000000000000000000348,ZK,8,120
+USD+,USD,RedStone Core,No,No,RedstoneCoreOracle,N/A,0x0000000000000000000000000000000000000348,USD+,8,120
+SolvBTC_BNB,USD,RedStone Core,No,No,RedstoneCoreOracle,N/A,0x0000000000000000000000000000000000000348,SolvBTC_BNB,8,120
+SolvBTC_BNB,BTC,RedStone Core,No,No,RedstoneCoreOracle,N/A,0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB,SolvBTC_BNB/BTC,8,120
+SolvBTC_BNB,BTC_TWAP_60,RedStone Core,No,No,RedstoneCoreOracle,N/A,0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB,SolvBTC_BNB/BTC-TWAP-60,8,120
+AR,USD,RedStone Core,No,No,RedstoneCoreOracle,N/A,0x0000000000000000000000000000000000000348,AR,8,120
+SolvBTC.ENA,USD,RedStone Core,No,No,RedstoneCoreOracle,N/A,0x0000000000000000000000000000000000000348,SolvBTC.ENA,8,120
+SolvBTC.ENA,BTC,RedStone Core,No,No,RedstoneCoreOracle,N/A,0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB,SolvBTC.ENA/BTC,8,120
+SolvBTC.ENA,BTC_TWAP_60,RedStone Core,No,No,RedstoneCoreOracle,N/A,0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB,SolvBTC.ENA/BTC-TWAP-60,8,120
+SEI,USD,RedStone Core,No,No,RedstoneCoreOracle,N/A,0x0000000000000000000000000000000000000348,SEI,8,120
+ZKL,USD,RedStone Core,No,No,RedstoneCoreOracle,0xfc385a1df85660a7e041423db512f779070fcede,0x0000000000000000000000000000000000000348,ZKL,8,120
+mETH,ETH_FUNDAMENTAL,RedStone Core,Yes,Yes,RedstoneCoreOracle,0xd5F7838F5C461fefF7FE49ea5ebaF7728bB0ADfa,0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2,mETH_FUNDAMENTAL,8,120
+ISEI,USD,RedStone Core,No,No,RedstoneCoreOracle,N/A,0x0000000000000000000000000000000000000348,ISEI,8,120
```

Comment:
CSVs should always have trailing newlines

Reply:
they are downloaded straight from the google sheets like that. it appears that google doesn't add the trailing newlines